pom.xml

```diff
@@ -139,7 +139,6 @@
                 <groupId>joda-time</groupId>
                 <artifactId>joda-time</artifactId>
                 <version>2.2</version>
-                <scope>test</scope>
```

Comment:
You can see the error form Inteliji when trying to make project. 
Those libraries are only available in the test scope without this change.

![screenshot from 2016-10-28 16 50 39](https://cloud.githubusercontent.com/assets/12660801/19810673/b3e17ffa-9d2e-11e6-89c3-066c4feadf67.png)